src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arr = sourceString.split(';');
+  const result = {};
```

Comment:
The variable name `result` is not descriptive. It's better to use a name that describes the role of the variable, like `stylesObject`.